db/migrate/20160708182331_remove_replication_worker_settings.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveReplicationWorkerSettings < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    say_with_time("Removing replication worker settings") do
+      SettingsChange.where("key like ?", "/workers/worker_base/replication_worker%").delete_all
```

Comment:
Minor but capitalize LIKE.  Also, I *think* there's a way to do this without sql fragments with AR